CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
If you're not planning on linking to anything you shouldn't need to include the brackets.

DWIK - we've been using a wiki page for our changelogs for the web services since it's easier to maintain

Reply:
Good catch. My hope is that since we are using git tags, unlike our services, this will be easier than a wiki for this project.